moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryMap.js

```diff
@@ -53,7 +53,7 @@ const ProjectSummaryMap = ({ data }) => {
           ref={mapRef}
           initialViewState={initialViewState}
           style={{ width: "100%", height: "60vh" }}
-          mapStyle={basemaps.streets.mapStyle}
+          mapStyle={basemaps[basemapKey].mapStyle}
```

Comment:
this is a small fix that I snuck it. The satellite layer in the summary map is working just like in the components map in production, but, in the deploy preview, the layer wouldn't toggle to satellite.